inc/config.class.php

```diff
@@ -157,6 +157,11 @@ public function prepareInputForAdd($input) {
       return $input;
    }
 
+   /**
+    * Permits to hide fields
+    * @param array $fields
+    * @return array $fields
```

Comment:
no need to mention $fields in return, prefer a short description of the returned data